src/main/kotlin/net/maizegenetics/phgv2/agc/AnnotateFastas.kt

```diff
@@ -125,14 +126,10 @@ class AnnotateFastas : CliktCommand(help = "Annotate FASTA files with sample nam
                 // written to the outputDir
                 // if  the line starts with >, append " sampleName=${sampleName}" to the line
 
-                // if file was previously compressed, we compress the new file.
-                // Strip off the .gz extension before writing the new non-compressed file.
-                // it wil be added when we re-compress the file.
-                // The file names will either be <sampleName>.fa or <sampleName>.gz
-                // When the file is compressed, we only add .gz as an extension as AGC
-                // takes everything before the last period as the sample name and we
-                // want the sample name to look like "B73" or "CML247" and not "B73.fa" or "CML247.fa"
-                val justName = if (fastaFile.endsWith(".gz")) sampleName else "${sampleName}.fa"
+                // Previously if the original file was previously compressed, we compressed the new file.
+                // We no longer do this, as the align-assemblies step requires uncompressed fasta files.
+                // The file names will  be <sampleName>.fa
+                val justName =  "${sampleName}.fa"
```

Comment:
What if the incoming file is already compress?

Reply:
This code will never compress the output fastas as they are used in the align-assemblies step, and anchorwave does not handle compressed files.  I didn't think of that when I originally wrote the code - I was only thinking of AGC which can handle the compressed files.